apps/site/assets/ts/schedule/components/schedule-finder/TableRow.tsx

```diff
@@ -61,6 +65,9 @@ const BusTableRow = ({
         onClick={onClick}
         onKeyPress={e => handleReactEnterKeyPress(e, onClick)}
       >
+        <td className="schedule-table__td--tiny">
+          <strong>{isSchoolTrip && "S"}</strong>
```

Comment:
If it isn't a school trip, this creates an extra table data element with an empty strong, we probably shouldn't render the `<td>` unless we need it.

Do we want to add some sort of hidden label here for SR that says "School trip"? I'm not sure what we do for this iteration for sighted users to know what "S" means.

Reply:
re: accessibility, will update -- makes perfect sense.

re: the table, it's a little tricky, because if any one of the trips happens to be a school trip, then they all need a `<td>` cell and header. Are you thinking to check if any trip is a school trip, and if not, don't generate the column?